recipe/meta.yaml

```
@@ -31,21 +30,24 @@ requirements:
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - ipython >=5.0                          # [build_platform != target_platform]
     - jupyter_client <7.0                    # [build_platform != target_platform]
```

```suggestion
    - jupyter_client <8.0                    # [build_platform != target_platform]
```

This constraint seems to be wrong. This  was probably added before jupyter_client 7 was stable, but now it shouldn't be necessary.